### PR TITLE
Fix #2669: profile name aligned properly

### LINKED ISSUE #2669: Profile name is not properly aligned
**Describe the bug**
username is not properly aligned on the profile page.

**To Reproduce**
1. click on the navigation drawer icon
2. Click on the profile picture
3. See the error

**Expected behavior**
The username should be correctly aligned in the center.

**Screenshots**

Screenshot_20210209-141125

**Device**
- Device/emulator being used: Real device Oneplus 7
- SDK version 29

### PR DESCRIPTION
**Pull request by yashraj-01**
## Explanation

Fixes #2669. Profile name is aligned properly in `profile-progress-header` in `layout` i.e. for `mobile-portrait` screen.

These are the updated screenshots.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by yashraj-01** [comment]
@rt4914 I suppose failing of the 3rd test is not a problem since I was told that it is a flake. Am I right?

File: app/src/main/res/layout/profile_progress_header.xml [hunk 1]
```diff
@@ -47,6 +47,7 @@
       android:layout_height="wrap_content"
       android:layout_marginTop="12dp"
       android:fontFamily="sans-serif-medium"
+      android:gravity="center"
       android:text="@{viewModel.profile.name}"
```
Comment (prayutsu) [inline app/src/main/res/layout/profile_progress_header.xml:51]: Is it already centered for other landscape and tablet layouts?
Please also add the screenshots for landscape and tablet layouts.

**Review by prayutsu — CHANGES_REQUESTED** [review]
@yashraj-iitr The PR Looks good, just update the other layout files if needed and add the screenshots in the PR description.

**Comment by prayutsu** [comment]
@yashraj-iitr Yes, the third check fails due to a flaky test case.
Also, please note that if your PR is ready for review, then you can add a comment - `@{Reviewer username} PTAL`.
So, that the reviewer can understand that the PR is ready for review.

File: app/src/main/res/layout/profile_progress_header.xml (hunk 1 above)
Comment (yashraj-01) [reply app/src/main/res/layout/profile_progress_header.xml:51]: as discussed with @rt4914  earlier in the issue, the profile need not be centre aligned in other layouts i.e `layout-land`, `layout-sw600dp-port` and `layout-sw600dp-land`. So, I did not change them.

**Comment by yashraj-01** [comment]
> @yashraj-iitr Yes, the third check fails due to a flaky test case.
> Also, please note that if your PR is ready for review, then you can add a comment - `@{Reviewer username} PTAL`.
> So, that the reviewer can understand that the PR is ready for review.

Got it. I'll be following this from now on. Thanks.

File: app/src/main/res/layout/profile_progress_header.xml (hunk 1 above)
Comment (prayutsu) [reply app/src/main/res/layout/profile_progress_header.xml:51]: Alright then, you can resolve this conservation.

**Review by prayutsu — APPROVED** [review]
LGTM!!, Thanks @yashraj-iitr

File: app/src/main/res/layout/profile_progress_header.xml (hunk 1 above)
Comment (yashraj-01) [reply app/src/main/res/layout/profile_progress_header.xml:51]: okay. thanks.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Comment by rt4914** [comment]
@BenHenning I have re-run all jobs twice and everytime atleast one of the checks is failing. Just assigning it to you if some check still fails.

**Pull request merged**